src/main/java/org/broadinstitute/hellbender/utils/test/TestResources.java

```diff
@@ -0,0 +1,131 @@
+package org.broadinstitute.hellbender.utils.test;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.broadinstitute.hellbender.utils.GenomeLoc;
+import org.broadinstitute.hellbender.utils.GenomeLocParser;
+import org.broadinstitute.hellbender.utils.fasta.CachingIndexedFastaSequenceFile;
+import org.testng.Assert;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Class containing helper methods and resolved directories for test resources.
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public final class TestResources {
```

Comment:
Instead of extracting a `TestResources` class like this, it would be much less invasive/disruptive to instead create a `GATKBaseTest` subclass of `BaseTest` with the references to GATK-specific test data. That way, you won't have to change every reference to every file -- you'll just have to change `extends BaseTest` to `extends GATKBaseTest`